firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -6667,6 +6667,19 @@ extension String {
         tableName: nil,
         value: nil,
         comment: "See http://mzl.la/1Qtkf0j")
+
+    public struct RemoteTabPanel {
+        public static let CloseTab_SendingNotification_title = MZLocalizedString(
+            key: "CloseTab.SendingNotification.title.v133",
+            tableName: "RemoteTabPanel",
+            value: "Tab closed",
+            comment: "Text in notification after sending close remote tab command.")
+        public static let CloseTab_UndoButton_title = MZLocalizedString(
+            key: "CloseTab.UndoButton.title.v133",
+            tableName: "RemoteTabPanel",
+            value: "Undo",
+            comment: "Label for undo button in notification after sending close remote tab command.")
+    }
```

Comment:
I actually don't think you need this Undo string -- we currently have a localized string already for the "Undo" we show in toasts. 😄 

Here is the string we have already in the file for your reference:
```
    public static let UndoString = MZLocalizedString(
        key: "Toasts.Undo",
        tableName: nil,
        value: "Undo",
        comment: "Label for button to undo the action just performed")
```